src/wooden-tree/defaultStore.js

```diff
@@ -0,0 +1,52 @@
+import Tree, { ActionTypes } from './';
+
+const actionMapper = {
+  [ActionTypes.EXPANDED]: Tree.nodeExpanded,
+  [ActionTypes.DISABLED]: Tree.nodeDisabled,
+  [ActionTypes.SELECTED]: Tree.nodeSelected,
+  [ActionTypes.CHILD_NODES]: Tree.nodeChildren,
+  [ActionTypes.LOADING]: Tree.nodeLoading,
```

Comment:
You should add `CHECKED` and `ADD_NODES` in here as well, so we don't have to keep this mapping on two places.